native/logik_simulation/src/ffi.rs

```diff
@@ -36,21 +36,21 @@ pub extern "C" fn exit(data: *mut Data) {
 }
 
 #[no_mangle]
-pub extern "C" fn add_subnet(data: *mut Data, id: usize) -> bool {
+pub extern "C" fn add_subnet(data: *mut Data, id: i32) -> bool {
     let data = unsafe { &mut *data};
     
-    data.add_subnet(id)
+    data.add_subnet(id as usize)
 }
 
 #[no_mangle]
-pub extern "C" fn remove_subnet(data: *mut Data, id: usize) -> bool {
+pub extern "C" fn remove_subnet(data: *mut Data, id: i32) -> bool {
     let data = unsafe { &mut *data};
     
-    data.remove_subnet(id)
+    data.remove_subnet(id as usize)
```

Comment:
Same thing here. If subnet ID:s in the datastructure were `i32`, there would be no need for a cast.